packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -243,9 +243,17 @@ function makeDeltaPoller_Notifications(store: UmbrellaStore<BaseMetadata>) {
     // in the `then` body, check again if the number of subscribers if more than 0, and only then
     // if those conditions hold, start the poller
     poller.enable(pollerSubscribers > 0);
```

Comment:
And I assume we'll want to remove this line here, right?

Reply:
Yes! Thanks for catching this!